src/main/resources/default/kb/en/system/book.html.pasta

```diff
@@ -0,0 +1,5 @@
+<k:chapter code="HDJEK" lang="en" priority="900" title="System Manual" permissions="flag-system-tenant" parent="00000">
```

Comment:
Frage: Wo kommen diese Codes her? Werden die von Jupiter generiert oder wählen wir die?

Reply:
Yep the idea is to generate random 5 char codes - maybe we even add a helper command or page for this to simplify this task.